src/_posts/2019-04-16-containers-with-docker.md

```diff
@@ -252,7 +252,8 @@ Finally, we can run the application by passing the `CMD` to start
 
 ```Dockerfile
 FROM adoptopenjdk/openjdk11-openj9
-WORKDIR /realworldCOPY . /realworld
+WORKDIR /realworld
+COPY . /realworld
```

Comment:
I would do `COPY . .` so that you don't have to define the working directory in two places.

Reply:
I'm just fixing the typo on the original code snippet, maybe Andre could intervene on that?

Reply:
`COPY . .` will also work in this case. I don't mind having it.

Reply:
IMHO, I think that keeping it the way you initially wrote it documents your progress as well.